sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -1068,3 +1068,11 @@ INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_re
 INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_release_date`, `load_filename`, `load_checksum`) VALUES
 ( 'CQM_VALUESET', 'NIH_VSAC', '2021-05-06', 'ep_ec_eh_cms_20210506.xml.zip', '6455da86e269edb6d33288e72b467373');
 #EndIf
+
+#IfMissingColumn form_encounter encounter_type_code
+ALTER TABLE `form_encounter` ADD `encounter_type_code` VARCHAR(31) NULL DEFAULT NULL, ADD `encounter_type_description` TEXT NULL;
+#EndIf
+
+#IfMissingColumn users billing_facility
+ALTER TABLE `users` ADD `billing_facility` TEXT NULL, ADD `billing_facility_id` INT(11) NOT NULL DEFAULT '0';
```

Comment:
You prob have this but don't forget this in `database.sql`

Reply:
Thanks Steven, I did forget! will get next push. y'all keep an eye on me....